conn.go

```diff
@@ -258,6 +253,33 @@ func (c *Conn) List() (exports []string, err error) {
 	return exports, nil
 }
 
+func (c *Conn) StartTLS(config *tls.Config) error {
+	if state := c.state(); state != connectionStateOptions {
+		return errNotOption
+	}
+
+	err := requestOption(c.conn, &startTLSRequest{})
+	if err != nil {
+		return err
+	}
+
+	reply, err := readOptionReply(c.conn)
+	if err != nil {
+		return err
+	}
+
+	if reply.Type != nbdproto.REP_ACK {
+		return errors.New("server did not reply with error or ACK")
```

Comment:
I'm not sure how this works, but would it be useful to mention what the reply was if it wasn't one of the expected ones?

Reply:
I think it could be useful debugging information, but none of the other Conn methods do this and I'd rather just add it for all of them in a separate PR than just 1 of them.